### ISSUE TITLE
Tasks created from backup have visual artifacts

### ISSUE THREAD

**Issue by Klazkin** [opencv/cvat#7688]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://opencv.github.io/cvat/docs/)

### Steps to Reproduce

Occurs only with certain videos, is reproducible on both self-hosted and https://app.cvat.ai/.

1. Download [this](https://drive.google.com/file/d/1wDtk85fWuelzreXI6YIdWpisc2iimSVm/view?usp=sharing) video. ([original source](https://twitter.com/UnusualVideos/status/1138350999448371200)) 
1. Go to Tasks tab.
2. From the dropdown select the  "Create a new task".
3. Select the downloaded file from step 1.
  a. Verify the video is first displaying as intended in the job for the task.
![Screenshot_20240327_164158](https://github.com/opencv/cvat/assets/44501483/7d043de8-4d9d-4a46-9561-afc39620e20e)
4. Return to the Tasks tab.
5. Locate the task added in step 3. and from actions select "Backup Task".
6. In Tasks tab, select "Create from backup" option in the dropdown.
7. Select the backup created in step 5.
8. If you go to the job for the task you will now see this:
![Screenshot_20240327_164115](https://github.com/opencv/cvat/assets/44501483/6c7b69a2-2c38-4a8e-bd12-bb0b16eef3d1)

Alternatively, the same issue also occurs when backing up the whole project and re-importing it from backup.
This only happens with certain videos, and it is always the same videos that cause this problem.

Viewing the video with a player from the backup .zip does not cause any issues.
If I export the task as a dataset, and save the images, then the images also do not have any issues with them (i.e. they do not have any visual artifacts)

![Screenshot_20240327_165239](https://github.com/opencv/cvat/assets/44501483/59c9465b-78be-43c0-a2e9-08ccffdf1dff)

### Expected Behavior

The video preview of a task created form backup should look the same as in the original.

### Possible Solution

I assumed it was something do to with the fps (based on the way it looks), but when adjusting the fps of the video using ffmpeg it did not seem to help. Specifically I used:

`fmpeg -i input_29fps.mp4 -filter:v fps=30 output_30fps.mp4`

### Context

I was trying to back up my project before making some important changes and also to re-import it within another Organization. 

In case it is relevant, these videos are from twitter and about 20-40% of them have this issue.

FFMPEG Video Metadata
```
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'pfEmPmGkTmg4J_n3.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 1
    compatible_brands: isomiso4
    creation_time   : 2024-03-16T04:42:55.000000Z
  Duration: 00:00:24.01, start: 0.000000, bitrate: 356 kb/s
  Stream #0:0[0x1](und): Video: h264 (Main) (avc1 / 0x31637661), yuv420p(progressive), 360x360, 320 kb/s, 29.97 fps, 29.97 tbr, 30k tbn (default)
    Metadata:
      creation_time   : 2019-06-11T07:43:17.000000Z
      handler_name    : Vireo Eyes v2.5.3
      vendor_id       : [0][0][0][0]
      encoder         : AVC Coding
  Stream #0:1[0x2](und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 32 kb/s (default)
    Metadata:
      creation_time   : 2019-06-11T07:43:17.000000Z
      handler_name    : Vireo Ears v2.5.3
      vendor_id       : [0][0][0][0]
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> wrapped_avframe (native))
  Stream #0:1 -> #0:1 (aac (native) -> pcm_s16le (native))
Press [q] to stop, [?] for help
Output #0, null, to 'pipe:':
  Metadata:
    major_brand     : isom
    minor_version   : 1
    compatible_brands: isomiso4
    encoder         : Lavf60.16.100
  Stream #0:0(und): Video: wrapped_avframe, yuv420p(progressive), 360x360, q=2-31, 200 kb/s, 29.97 fps, 29.97 tbn (default)
    Metadata:
      creation_time   : 2019-06-11T07:43:17.000000Z
      handler_name    : Vireo Eyes v2.5.3
      vendor_id       : [0][0][0][0]
      encoder         : Lavc60.31.102 wrapped_avframe
  Stream #0:1(und): Audio: pcm_s16le, 44100 Hz, mono, s16, 705 kb/s (default)
    Metadata:
      creation_time   : 2019-06-11T07:43:17.000000Z
      handler_name    : Vireo Ears v2.5.3
      vendor_id       : [0][0][0][0]
      encoder         : Lavc60.31.102 pcm_s16le
[out#0/null @ 0x646a1cd2ecc0] video:337kB audio:2068kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
frame=  719 fps=0.0 q=-0.0 Lsize=N/A time=00:00:23.98 bitrate=N/A speed= 299x 
```

### Environment

```Markdown
I'll skip this since I was able to reproduce it both on https://app.cvat.ai/ and my own self hosted CVAT.
```

**Comment by bsekachev** [comment]
Try this fix https://github.com/opencv/cvat/pull/7583 if installed offline.
On app.cvat.ai it will be deployed early next week.

**Comment by Klazkin** [comment]
Looks like it could work, thank you. I'll be able to verify it on Monday.

**Comment by Klazkin** [comment]
> Try this fix #7583 if installed locally. On app.cvat.ai it will be deployed early next week.

I tried the fix and it indeed works. Thank you and sorry for creating a duplicate issue.
